integrationtest/src/main/scala/io/gearpump/integrationtest/kafka/NumericalDataProducer.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gearpump.integrationtest.kafka
+
+import java.util.Properties
+
+import com.twitter.bijection.Injection
+import io.gearpump.util.LogUtil
+import org.apache.kafka.clients.producer.{ProducerRecord, KafkaProducer}
+import org.apache.kafka.common.serialization.{ByteArraySerializer, StringSerializer}
+
+class NumericalDataProducer(topic: String, bootstrapServers: String) {
+  private val LOG = LogUtil.getLogger(getClass)
```

Comment:
Better the same logger as other test routines use, if there is no strong reason to import `io.gearpump.util.LogUtil`. Add one line before and after class body declaration.